phpseclib/Crypt/Rijndael.php

```diff
@@ -755,7 +755,7 @@ function _subWord($word)
     {
         static $sbox;
         if (empty($sbox)) {
-            list(,,,, $sbox) = $this->_getTables();
+            list(, , , , $sbox) = $this->_getTables();
```

Comment:
This (or something similar) was previously claimed at #777 and apparently turned out to be wrong.